rudder-core/src/main/scala/com/normation/cfclerk/domain/RegexConstraint.scala

```diff
@@ -53,14 +53,14 @@ case class RegexConstraint(pattern: String, errorMsg: String) extends HashcodeCa
 
   val compiled = Pattern.compile(pattern)
 
+  val variablePattern = Pattern.compile(".*\\$\\{.*\\}.*")
```

Comment:
i fear it may be a bit too eager

Reply:
well, actually, i can't see why it should be an issue

Reply:
Yes we don't want to analyze it deeply do it does not need to be perfect matches for cases i tested ... (one var, two vars, var inside a var ... )

Reply:
I updated the regexp so it's less eager (added ? after * so it matches as less possible)